src/services/codefixes/importFixes.ts

```diff
@@ -620,4 +666,73 @@ namespace ts.codefix {
         // Need `|| "_"` to ensure result isn't empty.
         return !isStringANonContextualKeyword(res) ? res || "_" : `_${res}`;
     }
+
+    function createLazyPackageJsonDependencyReader(fromFile: SourceFile, host: LanguageServiceHost) {
+        const packageJsonPaths = findPackageJsons(getDirectoryPath(fromFile.fileName), host);
+        const dependencyIterator = readPackageJsonDependencies();
+        let seenDeps: Map<true> | undefined;
+        function *readPackageJsonDependencies() {

```

Comment:
probably would be easier to read if this were moved outside the containing function and took its two (I think?) closed-over variables as parameters.